tools/coverage.sh

```diff
@@ -9,13 +9,13 @@ mkdir -p build.cov
 cd build.cov/
 echo cmake -G Ninja -D REALM_COVERAGE=ON -D CMAKE_CXX_FLAGS=-g -DREALM_MAX_BPNODE_SIZE=${MAX_NODE_SIZE} ..
 cmake -G Ninja -D REALM_COVERAGE=ON -D CMAKE_CXX_FLAGS=-g -DREALM_MAX_BPNODE_SIZE=${MAX_NODE_SIZE} ..
-ninja realm-tests
+ninja
 cd ${PROJECT_DIR}
 if [ ! -f coverage-base.info ]; then
   lcov --no-external --capture --initial --directory . --output-file ./coverage-base.info
 fi
 cd build.cov/test/
```

Comment:
Should it not be just `cd build.cov`?

Reply:
It works like this, but I don't know what's more appropriate.